README.md

```diff
@@ -4,7 +4,7 @@
 [![Docs status](https://readthedocs.org/projects/weas-widget/badge)](http://weas-widget.readthedocs.io/)
 [![Unit test](https://github.com/superstar54/weas-widget/actions/workflows/ci.yml/badge.svg)](https://github.com/superstar54/weas-widget/actions/workflows/ci.yml)
 
-A widget to visualize and edit atomistic structures in Jupyter Notebook. It uses [WEAS](https://github.com/superstar54/weas) (Web Environment For Atomistic Structure) in the backend.
+A widget to visualize and edit atomic structures in Jupyter Notebooks. It uses [WEAS](https://github.com/superstar54/weas) (Web Environment For Atomistic Structure) in the backend.
```

Comment:
`atomic` structure is more used to describe the structure of an atom, e.g., nuclear, electron. `atomistic` structure is the combination of atoms, like molecule and crystal. I think `atomistic` is the better word here.

Reply:
That's interesting - so far I remember coming across "atomistic" only in conjunction with very specific nouns, such as "atomistic simulations", but you are right that results on Google for "atomic structure" are dominated by references to the internal structure of the atom.

I'm not sure whether this reflects the frequency of use in the scientific literature, it could also be that the frequency of visits/links is higher for topics that are also taught at school.

If you search on Google scholar instead, I would say the results already look different.
For example, the number of results for `"atomic structure" dft` are higher (98k) than the results for `"atomistic structure"` (13.8k)

https://scholar.google.com/scholar?hl=en&as_sdt=0%2C5&q=%22atomic+structure%22+dft&btnG=
https://scholar.google.com/scholar?hl=en&as_sdt=0%2C5&q=%22atomistic+structure%22&btnG=

This is a detail, of course, and it does not really matter - but it was interesting to learn about.

Reply:
Thanks for the detail. It's interesting that `atomic structure` is used much more than `atomistic structure` in Google Scholar. I also updated the docs for consistency.